corpus/codeblock.txt

```diff
@@ -114,15 +126,35 @@ h1-headline *foo*
         (line)
         (line)))
     (line
-      (word))
+      (h1
+        (word)
+        (tag
+          (word))))
```

Comment:
h1/h2/h3 are no longer top-level elements, they are now contained by `line`. This is necessary because implicitly-terminated `codeblock` (without `<`) consumes blank lines (because codeblocks can have blank and empty lines), so there is no way for `block` to know when to terminate and allow other top-level elements to start.